guides/mn-application-aws-lambda-graalvm/mn-application-aws-lambda-graalvm.adoc

```diff
@@ -22,7 +22,7 @@ The generated application contains a `HomeController`. It responds to POST reque
 
 source:HomeController[]
 
-callout:controller[arg0=/]
+* The class is defined as a controller with the @api@/io/micronaut/http/annotation/Controller.html[@Controller] annotation mapped to the path `/`.
```

Comment:
why are you replacing the callout with the sentence? Please, use the callout to avoid having to maintain the same copy in multiple places.

Reply:
leave it with the callout even if it displays a 1 which is not in the code.